business/istio_config.go

```diff
@@ -239,3 +226,23 @@ func (in *IstioConfigService) DeleteIstioConfigDetail(api, namespace, resourceTy
 	err = in.k8s.DeleteIstioObject(api, namespace, resourceType, name)
 	return
 }
+
+func getUpdateDeletePermissions(k8s kubernetes.IstioClientInterface, namespace, objectType string) (bool, bool) {
+	var canUpdate, canDelete bool
+	if api, ok := resourceTypesToAPI[objectType]; ok {
+		ssars, permErr := k8s.GetSelfSubjectAccessReview(namespace, api, objectType, []string{"update", "delete"})
+		if permErr == nil {
+			for _, ssar := range ssars {
+				switch ssar.Spec.ResourceAttributes.Verb {
```

Comment:
This field is a pointer:
```
ResourceAttributes *ResourceAttributes
```
So, even probably it's always filled, I think is safer to check a ResourceAttributes != nil first to avoid cornercases.

Reply:
fixed